src/modules/commons/components/checkbox/Checkbox.js

```diff
@@ -5,10 +5,12 @@ import { html } from 'lit-html';
 import { TEST_ID_ATTRIBUTE_NAME } from '../../../../utils/markup';
 import { MvuElement } from '../../../MvuElement';
 import css from './checkbox.css';
+import { classMap } from 'lit-html/directives/class-map.js';
 
 const Update_Disabled = 'update_disabled';
 const Update_Checked = 'update_checked';
 const Update_Title = 'update_title';
+const Update_Type = 'update_type';
 
 /**
```

Comment:
Let us take this opportunity to update the documentation to the current standard.
```js
 @fires  change when the ...
 @property ...
```